spring-cloud-app-broker-core/src/main/java/org/springframework/cloud/appbroker/service/WorkflowServiceInstanceBindingService.java

```diff
@@ -139,7 +139,7 @@ else if (isRouteBindingRequest(request)) {
 
 	private boolean isAppBindingRequest(CreateServiceInstanceBindingRequest request) {
 		return request.getBindResource() != null
-			&& StringUtils.isEmpty(request.getBindResource().getRoute());
+			&& !StringUtils.hasLength(request.getBindResource().getRoute());
```

Comment:
What's the reason for this change? I think `isEmpty` is easier to parse than negated `hasLength`.

Reply:
Oh, now I see I was looking at the old version of `StringUtils` and `isEmpty` is deprecated as of 5.3.

Reply:
Yeah, I prefer the `isEmpty` API as well. It's very clear to me. I guess there were reasons for deprecating it.

Reply:
You could use `ObjectUtils.isEmpty` instead.

Reply:
Good point. I'm good either way, but the isEmpty is still more clear to me.